frontend/src/pages/pipelines/global/modelCustomization/ModelCustomizationForm.tsx

```diff
@@ -69,11 +70,17 @@ const ModelCustomizationForm: React.FC = () => {
       endpoint: '',
       modelName: '',
     },
+    runType: { value: '' },
+    hyperparameters: {},
   });
 
   const validation = useValidation(data, modelCustomizationFormSchema);
   const navigate = useNavigate();
 
+  React.useEffect(() => {
+    setData('hyperparameters', filterHyperparameters(ilabPipelineVersion));
```

Comment:
this function returns a parameterKF type. we want to just be storing the actual value here not the stuff from the pipeline input param schema